tests/logictest/suites/base/06_show/06_0003_show_settings

```diff
@@ -36,5 +37,6 @@ SHOW SETTINGS LIKE 'enable%';
 enable_async_insert 0 0 SESSION Whether the client open async insert mode, default value: 0 UInt64
 enable_new_processor_framework 1 1 SESSION Enable new processor framework if value != 0, default value: 1 UInt64
 enable_planner_v2 1 1 SESSION Enable planner v2 by setting this variable to 1, default value: 1 UInt64
+enable_prewhere  1  1  SESSION  Enable prewhere optimization by setting this variable to 1, default value: 1  UInt64
```

Comment:
Maybe we should not introduce a new setting.

1. we can control it inside the code without exporting a new setting to users.
2. we will have an optimizer rule setting to control whether optimizers could be enabled or not in the future.

Reply:
I see.